TestON/drivers/common/cli/emulator/mininetclidriver.py

```diff
@@ -1377,20 +1377,30 @@ def stopNet( self, fileName = "", timeout=5):
             response = main.FALSE
         return response
 
-    def arping( self, src, dest, destmac ):
-        self.handle.sendline( '' )
-        self.handle.expect( [ "mininet", pexpect.EOF, pexpect.TIMEOUT ] )
-
-        self.handle.sendline( src + ' arping ' + dest )
+    def arping( self, host="", ip="10.128.20.211" ):
+        """
+        Description:
+            Sends arp message from mininet host for hosts discovery
+        Required:
+            host - hosts name
+        Optional:
+            ip - ip address that does not exist in the network so there would
+                 be no reply.
```

Comment:
I think we only use it to force the host to send the arp message. 
Trying out both successful/unuccessful:
sudo arping 10.128.30.112 -c 1
ARPING 10.128.30.112

--- 10.128.30.112 statistics ---
1 packets transmitted, 0 packets received, 100% unanswered (0 extra)

sudo arping 10.128.30.11 -c 1
ARPING 10.128.30.11
60 bytes from 0e:03:8d:d9:52:18 (10.128.30.11): index=0 time=571.026 msec

--- 10.128.30.11 statistics ---
1 packets transmitted, 1 packets received,   0% unanswered (0 extra)


Also, we should add -c 1 to this string or it will continue as ping does